### PR TITLE
New home layout

### PR DESCRIPTION
**Pull request by lbulgarelli**
I've made changes to improve the home page layout.

Here's a summary:

- Removed container so we have full screen content
- Changed colors (badges, urls, ...)  so they are more friendly to the eyes
- Added padding to the badges
- Moved news and welcome to the left
- Changed search style to match nav bar
- Changed `New Content` to `Latest resources`

Here's the result:

Screen Shot 2019-03-18 at 10 24 53 AM

### CODE REVIEW

File: physionet-django/static/custom/css/form-control-input.css [hunk 1]
```diff
@@ -41,3 +41,7 @@ input::placeholder {
   color: #868e96;
   opacity: 1;
 }
+
+.btn-search{
```
Comment (tompollard) [inline physionet-django/static/custom/css/form-control-input.css]: Is this empty style here intentionally? If so, please could you explain its purpose?

File: physionet-django/templates/home.html [hunk 2]
```diff
@@ -14,58 +14,43 @@
...
               {{ news.content|safe }}
               <hr>
             {% endfor %}
-            <a class="btn btn-outline-dark" href="{% url 'news' %}">More News</a>
+            <div class="more"><a class="btn btn-outline-dark" href="{% url 'news' %}">More news</a></div>
           </div>
         </div>
+      </div> <!--Right Column --><div class="main-content">
+        <h1>Latest resources</h1><br>
+        <!-- <hr> -->
```
Comment (tompollard) [inline physionet-django/templates/home.html]: Unless it's there for a reason, the hr comment can just be removed.

**Comment by tompollard** [comment]
Thanks Lucas! It looks like a couple of your changes have conflicted with the dev branch, so please could you rebase and push again? I've added a couple of comments to the code, but my CSS isn't great so others might also want to take a look...

**Comment by tompollard** [comment]
Looks great, to me thanks Lucas! @elfeto any comments before we merge?

**Comment by elfeto** [comment]
If you move the Account button in the navbar to the left, so that is at the other side, and give a color to the magnifying glass then I am ok :)

**Comment by tompollard** [comment]
Great work, thanks Lucas.

**Pull request merged**